2023/README.md

```diff
@@ -1,8 +1,12 @@
 # Interop 2023
 
-Interop 2023 aims to make the web more interoperable in key areas, prioritized by web developer and user needs. The team is formed of browser vendors and other contributors to the web platform, and is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt).
+Interop 2023 is an effort to increase interoperability across browsers in key technical areas that are of high priority to web developers and for user needs. It's part of the [web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT) project — an automated test suite for web standards — and run by a team of representatives from companies that make substantial contributions to browser rendering engines.
 
-Similar to [Interop 2022](https://wpt.fyi/interop-2022), this will be a public metric based on the progress of selected focus areas and investigation efforts. The overall timeline for the planning process is:
+Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests selected will be continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the Interop 2023 Dashboard — displaying the percentage of passing tests in each chosen area, and an overall total score. The dashboard will also display scores for group progress on particular Investigation Efforts selected for the the Interop team to work on throughout the year.
+
+Interop 2023 is tightly focused on technology that is already specified in web standards. This is not a place to invent new web technology, nor solve complex problems that need web standards. That work should happen in the standards bodies. 
```

Comment:
```suggestion
Interop 2023 is tightly focused on technology that is already specified in web standards. This is not a place to invent new web technology: that work should happen in the standards bodies. 
```
Or I'm not sure quite what the second clause was trying to say that's different from the first. Maybe it was trying to make the distinction between really novel standards and major (non-interop motivated) changes to existing standards? If so perhaps "This is not a place to invent new web technology, nor make novel additions to existing specifications." (there are some edge cases there with investigate, but it probably doesn't matter).

Reply:
I like the simplification of the first clause. Did you mean to suggest a semicolon though? IIRC you suggested it in the other place:

> Interop 2023 is not a venue for specifying new features; that work happens in working groups within organizations such as W3C and WHATWG.

Reply:
Sure, a semicolon works.